cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/mongo/MongoBackendImpl.java

```diff
@@ -162,14 +176,33 @@ public void createCollection(String dbName, String collectionName, long collecti
         } // try catch
```

Comment:
From which sink is this `createCollection()` is being called? MongoSink or STHSink?